src/Data/Record/Extra.purs

```diff
@@ -175,6 +175,38 @@ class Applicative m <= SequenceRecord rl row row' m
   where
     sequenceRecordImpl :: RLProxy rl -> Record row -> m (Record row')
 
+class OrdRecord rl row
+  | rl -> row
+  where
+    ordRecordImpl :: RLProxy rl -> Record row -> Record row -> Ordering
+
+instance ordRecordCons ::
+  ( IsSymbol name
+  , Ord ty
+  , RowCons name ty trash row
+  , OrdRecord tail row
+  ) => OrdRecord (Cons name ty tail) row where
+  ordRecordImpl _ a b =
+    case compare valA valB of
+         EQ -> ordRecordImpl tailp a b
+         ordering -> ordering
+    where
+      namep = SProxy :: SProxy name
+      valA = get namep a
+      valB = get namep b
+      tailp = RLProxy :: RLProxy tail
+
+instance ordRecordNil :: OrdRecord Nil row where
+  ordRecordImpl _ _ _ = EQ
+
+ordRecord :: forall row rl
+   . RowToList row rl
+  => OrdRecord rl row
+  => Record row
+  -> Record row
+  -> Ordering
+ordRecord a b = ordRecordImpl (RLProxy :: RLProxy rl) a b
```

Comment:
minor nitpick, I guess this should really be called `compareRecord`?

Reply:
Good point. Will amend.